cms/standard_pages/models.py

```diff
@@ -1,13 +1,16 @@
-from typing import ClassVar
+from typing import TYPE_CHECKING, ClassVar
 
 from django.db import models
 from wagtail.admin.panels import FieldPanel, InlinePanel
 from wagtail.search import index
 
-from cms.core.blocks.stream_blocks import CoreStoryBlock
+from cms.core.blocks.stream_blocks import SectionStoryBlock
 from cms.core.fields import StreamField
 from cms.core.models import BasePage
 
+if TYPE_CHECKING:
+    pass
+
```

Comment:
this (and the `TYPE_CHECKING` import) is not needed

Reply:
Fixed as part of [1ea7ed](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/1ea7ed1e96e994acd51f46a43ee29a3956aa6c39)